checker/src/org/checkerframework/checker/index/samelen/SameLenAnnotatedTypeFactory.java

```diff
@@ -73,6 +77,42 @@ public QualifierHierarchy createQualifierHierarchy(MultiGraphFactory factory) {
         return new SameLenQualifierHierarchy(factory);
     }
 
+    @Override
+    public AnnotatedTypeMirror getAnnotatedTypeLhs(Tree tree) {
+        AnnotatedTypeMirror atm = super.getAnnotatedTypeLhs(tree);
+        if (tree.getKind() == Tree.Kind.VARIABLE) {
+            String varName = null;
+            Element elt = TreeUtils.elementFromDeclaration((VariableTree) tree);
+            if (elt.getKind() == ElementKind.LOCAL_VARIABLE
```

Comment:
Because of that, I'm not sure what you're asking for here - should `internalReprOfVariable` return a `String`? That seems strange. I'm not sure how to go from an `Element` to a `Receiver`, either.